databricks/koalas/utils.py

```diff
@@ -22,9 +22,6 @@
 import types
```

Comment:
@ueshin, I thought we can remove this entire file. Does that require some more works?